### PR TITLE
Added webupload, thunder, sun, quotes & usage plugins

### PR DESCRIPTION
**Pull request by shivamvijaywargi**
Included copyrights this time :)
 Included Sun, Thunder, and Quotes as separate plugins.

### CODE REVIEW

File: plugins/Quotes.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	".bleed - I’m not replying, but keep writing. I enjoy the way your words makes my eyes bleed.",
+	".life - Life’s good, you should get one.",
+	".repeat - No, you don’t have to repeat yourself. I was ignoring you the first time."
+)
+
+
+
+
+
+@userge.on_cmd("quote$", about="__Check yourself ;)__")
```
Comment (rking32) [inline plugins/Quotes.py]: this will conflict .quote 🤔

Comment (rking32) [reply plugins/Quotes.py]: can you change this command to like .quotes ?

Comment (shivamvijaywargi) [reply plugins/Quotes.py]: Yep, already removes that plugin.

Comment (rking32) [reply plugins/Quotes.py]: don't remove it
just change command

Comment (lacyboi) [reply plugins/Quotes.py]: But he did😂

File: plugins/Sun.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#
+# All rights reserved.
+
+
+from asyncio import sleep
+from collections import deque
+from userge import userge, Message
+
+
+@userge.on_cmd("sun$", about="__kensar sun animation__")
```
Comment (rking32) [inline plugins/Sun.py:15]:
```suggestion
@userge.on_cmd("sun$", about={'header': "kensar sun animation"})
```

File: plugins/Sun.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2020 by UsergeTeam@Github, < https://github.com/UsergeTeam >.
+#
+# This file is part of < https://github.com/UsergeTeam/Userge > project,
+# and is released under the "GNU v3.0 License Agreement".
+# Please see < https://github.com/uaudith/Userge/blob/master/LICENSE >
+#
+# All rights reserved.
```
Comment (rking32) [inline plugins/Sun.py:7]:
```suggestion
```
don't need this
feel free to add anything

File: plugins/Thunder.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2020 by UsergeTeam@Github, < https://github.com/UsergeTeam >.
+#
+# This file is part of < https://github.com/UsergeTeam/Userge > project,
+# and is released under the "GNU v3.0 License Agreement".
+# Please see < https://github.com/uaudith/Userge/blob/master/LICENSE >
+#
+# All rights reserved.
```
Comment (rking32) [inline plugins/Thunder.py:7]:
```suggestion
```

File: plugins/Thunder.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+
+from asyncio import sleep
+from collections import deque
+from userge import userge, Message
+
+
+@userge.on_cmd("thunder$", about="__kensar thunder animation__")
```
Comment (rking32) [inline plugins/Thunder.py:15]:
```suggestion
@userge.on_cmd("thunder$", about={'header': "kensar thunder animation"})
```

File: plugins/usage.py [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+# ================= CONSTANT =================
+Heroku = heroku3.from_key(Config.HEROKU_API_KEY)
+heroku_api = "https://api.heroku.com"
+HEROKU_APP_NAME = Config.HEROKU_APP_NAME
+HEROKU_API_KEY = Config.HEROKU_API_KEY
+# ================= CONSTANT =================
+
+
+@userge.on_cmd("usage", outgoing=True, about="__Get Dyno hours usage__")  # pylint:disable=E0602
```
Comment (rking32) [inline plugins/usage.py:15]:
```suggestion
@userge.on_cmd("usage", about={'header': "Get Dyno hours usage"})  # pylint:disable=E0602
```

File: plugins/webupload.py [hunk 7]
```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2020 by UsergeTeam@Github, < https://github.com/UsergeTeam >.
+#
+# This file is part of < https://github.com/UsergeTeam/Userge > project,
+# and is released under the "GNU v3.0 License Agreement".
+# Please see < https://github.com/uaudith/Userge/blob/master/LICENSE >
+#
+# All rights reserved.
```
Comment (rking32) [inline plugins/webupload.py:7]:
```suggestion
```

Comment (shivamvijaywargi) [reply plugins/webupload.py:7]: Aye Aye Captain..!

**Pull request merged**